packages/block-library/src/pullquote/editor.scss

```diff
@@ -35,12 +35,12 @@
 		font-size: 32px;
 	}
 
-	.wp-block-pullquote__citation {
+	footer {
```

Comment:
The switch to not using a class here makes the CSS simpler and easier to override, but it may cause minor styling issues for themes that were using this class to style blocks. I'm not sure how long the class has been around, so it may be that there are few or even no themes that actually use it. I could apply styles to both and add the class back into the markup and deprecate it, if desired.

I'd also appreciate some testing to make sure everything looks the way it should compared to `master`.